.github/workflows/cassandra-integration-tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Cassandra Integration Tests
```

Comment:
just a suggestion: call all workflows "Integration Tests", since there's also a distinct job name to uniquely identify each.

Reply:
I think GitHub actions actually group the workflows by name, not file. If you use the same name in different files, you might end up with a single workflow, no ?

Reply:
No, I was wrong: just tested by restarting one of the integration test jobs:

![image](https://user-images.githubusercontent.com/13387/100461145-a4b74a00-30c8-11eb-91cd-eeab83ef2dd6.png)